### PR TITLE
Display my applications

### LINKED ISSUE #1151: Implement My Profile - Integrations - My Applications list

### PR DESCRIPTION
**Pull request by eatonphil**
screen shot 2017-02-17 at 3 58 31 pm

Closes #1151

### CODE REVIEW

File: src/components/Feedback.js [hunk 1]
```diff
@@ -18,53 +20,48 @@ export default class Feedback extends Component {
...
-                <div>
-                  <input
-                    value={email}
-                    name="email"
-                    className="form-control"
-                    onChange={e => this.setState({ email: e.target.value })}
-                  />
-                </div>
+        <Card title="Submit Feedback">
+          <form
```
Comment (na3d) [inline src/components/Feedback.js]: Form?

File: src/components/Feedback.js [hunk 2]
```diff
@@ -18,53 +20,48 @@ export default class Feedback extends Component {
...
+            onSubmit={() => submitFeedback(this.state)}
+          >
+            <div className="form-group">
+              <div>
+                <label>Email:</label>
+              </div>
+              <div>
+                <input
```
Comment (na3d) [inline src/components/Feedback.js]: Input?

File: src/components/Feedback.js (hunk 1 above)
Comment (eatonphil) [inline src/components/Feedback.js]: Oh, good call!

File: src/components/Feedback.js (hunk 2 above)
Comment (eatonphil) [inline src/components/Feedback.js]: Wait a second, these are not part of the scope heh. I just updated the thing to use the card component.

Comment (na3d) [inline src/components/Feedback.js]: Same type of thing for other inputs.

File: scss/components/cards/Card.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    margin: 0 0 15px 0;
+
+    &-header {
+        margin-bottom: 10px;
+    }
+
+    &-title {
+        text-align: center;
+        overflow: hidden;
+        line-height: $form-field-height;
```
Comment (mparke) [inline scss/components/cards/Card.scss]: is this needed?

Comment (eatonphil) [inline scss/components/cards/Card.scss]: Good call, will check.

Comment (eatonphil) [inline scss/components/cards/Card.scss]: Dropped.

**Comment by eatonphil** [comment]
thank you!

**Pull request merged**